Add a Fail2ban Documentation (#488)
jellyfin/jellyfin-docs: general/networking/Fail2ban.md

---

> uid: Fail2ban-Jellyfin

Reviewer: The filename and UID should probably be lowercase.

title: Fail2ban
---

## Fail2ban

Fail2Ban is an intrusion prevention software framework that protects computer servers from brute-force attacks.

> Fail2Ban operates by monitoring log files (e.g. /var/log/auth.log, /var/log/apache/access.log, etc.) for selected entries and running scripts based on them.

Reviewer: ```suggestion
Fail2Ban operates by monitoring log files (e.g. /var/log/auth.log, /var/log/apache/access.log, etc.) for selected entries and running scripts based on their content.
```

> Jellyfin produces log that can be monitored by Fail2ban to prevent brute-force attacks on remote instance of jellyfin.

Reviewer: ```suggestion
Jellyfin produces logs that can be monitored by Fail2ban to prevent brute-force attacks on your machine.
```

### Requirements

* Jellyfin remotely accessible
* Fail2ban installed and running
* Knowing where the log for Jellyfin are stored : by default /var/log/jellyfin/

> ### Step One create a jail

Reviewer: ```suggestion
### Step One: Create a Jail
```

Reviewer: The other steps should also use a colon.

Reviewer 2: I'd also remove the capital letter from "One"

You need to create a jail for Fail2ban.
If you are on Ubuntu and use nano as editor, type :

```bash
sudo nano /etc/fail2ban/jail.d/jellyfin.local
```

> And add this to the file :

Reviewer: At minimum there should be no space before the colon, but full sentences are preferred ouside titles and headers.

Author: What can i say, french people loves spaces after colon :)

I made the changes.

Thank you for the modifications.
